### PR TITLE
Homework with multithreading.

### PR DESCRIPTION
**Pull request by Cherkashyn**

### CODE REVIEW

File: homework1/src/main/java/lesson9/BlockingQueue/Consumer.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    private BlockingQueue queue;
+
+    public Consumer(BlockingQueue queue) {
+        this.queue = queue;
+    }
+
+    @Override
+    public void run() {
+        try {
+            System.out.println(queue.take());
```
Comment (SergeyPasko) [inline homework1/src/main/java/lesson9/BlockingQueue/Consumer.java]: In one step - take three times. Ok

File: homework1/src/main/java/lesson9/BlockingQueue/Producer.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+public class Producer implements Runnable{
+    private BlockingQueue queue;
+
+    public Producer(BlockingQueue queue) {
+        this.queue = queue;
+    }
+
+    @Override
+    public void run() {
```
Comment (SergeyPasko) [inline homework1/src/main/java/lesson9/BlockingQueue/Producer.java]: You also set to blockingQueue three elements. Please, change to 10+. In this constructions I don't know - that you understand how it working

File: homework1/src/main/java/lesson9/StaticBlock/Counter.java [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    }
+
+    //method use a static synchronization - synchronization at the Class level
+    public static synchronized int staticIncrement(int value){
+        return staticNumber + value;
+    }
+
+    //method use synchronized block - synchronization at the Object level
+    public int synchronizedBlockIncrement(int value){
+        synchronized (this){
```
Comment (SergeyPasko) [inline homework1/src/main/java/lesson9/StaticBlock/Counter.java]: It isn't synchonized block. It is same to synchronized method

File: homework1/src/main/java/lesson9/BlockingQueue/Consumer.java (hunk 1 above)
Comment (Cherkashyn) [reply homework1/src/main/java/lesson9/BlockingQueue/Consumer.java]: I understood. I can use whlile here

**Pull request merged**